src/Helper/HasPages.php

```diff
@@ -4,19 +4,18 @@
 
 use eLife\ApiSdk\Collection\Sequence;
 use eLife\Journal\Pagerfanta\SequenceAdapter;
-use eLife\Patterns\ViewModel\Teaser;
 use GuzzleHttp\Promise\PromiseInterface;
 use Pagerfanta\Pagerfanta;
 use Symfony\Component\HttpFoundation\Request;
 use function GuzzleHttp\Promise\promise_for;
 
 trait HasPages
 {
-    final private function pagerfantaPromise(Sequence $sdkClient, int $page, int $perPage, string $convertTo = Teaser::class) : PromiseInterface
+    final private function pagerfantaPromise(Sequence $sdkClient, int $page, int $perPage, callable $map = null) : PromiseInterface
```

Comment:
Cause a few changes elsewhere since the assumption that they're not becoming teasers has been removed.